app/javascript/view/dictionary.js

```diff
@@ -52,7 +52,7 @@ OWSO.DictionariesEdit = (() => {
   }
 
   function onClickMarkAsFeedback() {
-    $('#variable_report_enabled').on('change', function(e) {
+    $('#variable_mark_as_report').on('change', function(e) {
```

Comment:
please consider giving a variable id/name same as what UI does, in e.g: UI is mark as feedback -> this should be a id/name that include 'mark_as_feedback'